### PR TITLE
Nemo/Changed services names.

### PR DESCRIPTION
**Pull request by nemodjuric**
Connects to #3917 

To verify check all 3 Monitor env and compare service names with services names in the component.

### CODE REVIEW

File: client/app/components/PerformanceDegradationBanner.jsx [hunk 1]
```diff
@@ -18,12 +18,11 @@ export default class PerformanceDegradationBanner extends React.Component {
     };
 
     this.dependencies = {
-      certification: ['BGS.FilenumberService', 'BGS.PoaService', 'VACOLS', 'VBMS', 'VBMS.FindDocumentSeriesReference'],
+      certification: ['BGS.AddressService', 'BGS.BenefitsService', 'BGS.ClaimantFlashesService', 'BGS.ClaimantGeneralInfoService', 'BGS.OrganizationPoaService', ' BGS.PersonFilenumberService', 'BGS.VeteranService', 'VACOLS', 'VBMS', 'VBMS.FindDocumentSeriesReference'],
```
Comment (amprokop) [inline client/app/components/PerformanceDegradationBanner.jsx]: does certification really depend on all these services?

Comment (nemodjuric) [reply client/app/components/PerformanceDegradationBanner.jsx]: Good question. I'll try to see if it's documented anywhere.

Comment (nemodjuric) [reply client/app/components/PerformanceDegradationBanner.jsx]: After digging around and trying to clarify some differences in naming between Monitor, Caseflow, ruby-bgs and BGS service itself, I found out that all of these services are used by Certification, except the one named "BGS.ClaimantGeneralInfoService".

Comment (nemodjuric) [reply client/app/components/PerformanceDegradationBanner.jsx]: As a future reference, here are the services and methods with all their versions used by Certification:
BGS Services used by Caseflow:
BGS.Service: api method(Caseflow/ruby-bgs)

OrgWebService: find_po_as_by_file_numbers(fetch_poa_by_file_number/find_poas_by_file_number)
PersonWebService: find_person_by_ssn (fetch_file_number_by_ssn/find_by_ssn)
BenefitClaimWebService: find_benefit_claim (get_end_products/find_by_vbms_file_number)
AddressWebService: find_all_ptcpnt_addrs_by_ptcpnt_id(find_address_by_participant_id/find_all_by_participant_id)
VeteranWebService: find_veteran_by_file_number(fetch_veteran_info/find_by_file_number)
ClaimantWebService: find_flashes(can_access?/can_access?)

**Pull request merged**